config/locales/fra/models.yml

```diff
@@ -326,6 +326,8 @@ fra:
           unvalid_amounts: "L’écriture doit être renseignée soit au débit, soit au crédit mais pas les deux en même temps."
         outgoing_parcel_item:
           greater_than_undelivered_quantity: "La quantité de produit %{product} à livrer dépasse la quantité non livrée (%{maximum}%{unit})"
+        purchase:
+          payment_delay_custom_validation_message: "Formats acceptés : fdm, fin du mois, %{values}"
```

Comment:
`fdm` and `fin de mois` should be in the interpolation too.

(With something like `values: TRANSLATIONS[...].... + ["fdm", "fin du mois"]` probably)

Also, `début du mois`, `ddm` and `avant` should be in the list too.